CIP-0100/README.md

```diff
@@ -162,6 +162,16 @@ We expect a rich ecosystem of CIPs to emerge defining different extensions, so t
 
 The rest of this document will provide a high level description of how these fields should be interpreted
 
+### HTML Tags
+
+For downstream tooling we define a reduced supported set of HTML tags which can be included in freefrom text fields.
+
+| Tag      | Description |
```

Comment:
Please people feel free to suggest more :)

Also is HTML the method we want to use, could use the markdown ones like `**bold text**`.

Reply:
> Also, why HTML over the much more industry standard markdown?

one justification: because there is one convergent definition of HTML while the "industry standard" of Markdown has produced a bunch of flavours with irreconcilable special features.  Those special features in Markdown come from special symbol behaviour, while in HTML it's delineated with tags which can be specifically enumerated herein.

Generally (and maybe even more relevant for blockchain histories), if I had to produce a richly formatted document to go into a time capsule to be readable and/or parseable in 100 years I would rather it be in HTML than any other format.

Reply:
I don't actually feel that that's the case, except in some extreme niche features; for the most part, people have standardized on github flavored markdown.

The reason I'd argue for GHMD over html, is that it's a lot easier for someone (especially non-technical users) to *write* in a textbox, and it's also a lot less confusing to read if you're looking at the raw data without the formatting applied.

Even if you know nothing about markdown, asterisks around a word are pretty intuitively indicative of emphasis, while `<b>Some word</b>` could be confusing to someone very non-technical.

Reply:
> The reason I'd argue for GHMD over html, is that it's a lot easier for someone (especially non-technical users) to write in a textbox, and it's also a lot less confusing to read if you're looking at the raw data without the formatting applied.

This is a good point, id be in favour of adopting this.

Reply:
@Ryun1 @Quantumplation yes, Markdown in hindsight I do think would be better.  I was thinking last night in terms of generality but today in terms of failure modes... mainly what happens if the markup _isn't rendered_.  As also suggested in https://github.com/cardano-foundation/CIPs/pull/748#discussion_r1462312205 the non-rendered Markdown is more readable than non-rendered HTML.